docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
 
+[[cluster-stats-api-response-body]]
+==== {api-response-body-title}
+
+`_nodes.total`::
+(integer)
+Total number of nodes in the cluster.
+
+`_nodes.successful`::
+(integer)
+Number of nodes that started succesfully.
+
+`_nodes.failed`::
+(integer)
+Number of nodes that failed to start.
```

Comment:
Missed one! This is the number of nodes that timed out or rejected the stats request, and if nonzero we also include the reason for the failure:

```
GET /_cluster/stats?filter_path=_nodes&timeout=1s

# {
#   "_nodes": {
#     "failures": [
#       {
#         "node_id": "hy7iAfvZStOy0C78T0BfJg",
#         "caused_by": {
#           "reason": "[node-4][127.0.0.1:9304][cluster:monitor/stats[n]] request_id [14552] timed out after [1009ms]",
#           "type": "receive_timeout_transport_exception"
#         },
#         "reason": "Failed node [hy7iAfvZStOy0C78T0BfJg]",
#         "type": "failed_node_exception"
#       }
#     ],
#     "successful": 4,
#     "total": 5,
#     "failed": 1
#   }
# }
```

Reply:
Thanks! Fixed with 23abdd43d45.